EstateManagement.Client/EstateClient.cs

```diff
@@ -249,6 +249,42 @@ public class EstateClient : ClientProxyBase, IEstateClient
             return response;
         }
 
+        public async Task DisableTransactionFeeForProduct(String accessToken,
+                                                          Guid estateId,
+                                                          Guid contractId,
+                                                          Guid productId,
+                                                          Guid transactionFeeId,
+                                                          CancellationToken cancellationToken)
+        {
+            String requestUri = this.BuildRequestUrl($"/api/estates/{estateId}/contracts/{contractId}/products/{productId}/transactionFees/{transactionFeeId}");
+
+            try
+            {
+                String requestSerialised = String.Empty;
+
+                StringContent httpContent = new StringContent(requestSerialised, Encoding.UTF8, "application/json");
+
+                // Add the access token to the client headers
+                this.HttpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
+
+                // Make the Http Call here
+                HttpResponseMessage httpResponse = await this.HttpClient.PostAsync(requestUri, httpContent, cancellationToken);
+
+                // Process the response
+                String content = await this.HandleResponse(httpResponse, cancellationToken);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'content'.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=5995655)